hack/test-go.sh

```diff
@@ -303,6 +309,9 @@ reportCoverageToCoveralls() {
   fi
 }
 
+# Start etcd
+kube::etcd::start
```

Comment:
Agree w/ @wojtekt; if at all possible, unit tests that need etcd should link in etcd and start an in-memory etcd, specific for that unit test.

Failing that, each test should individually start an etcd via exec (ugh; there's some work along these lines in the integration test).

A global etcd like this seems like it's asking for flaky tests, so I really prefer we don't do it unless there's no other way.